ui/src/pages/Feed.svelte

```diff
@@ -215,6 +222,7 @@
             </div>
             <input
               type="text"
+              id="feedGenerator"
```

Comment:
With svelte, instead of using `document.getElementById` you can instead bind the input to a variable, and use that variable. See here: https://svelte.dev/repl/76a65cdbd14f40a5818ab327c7b3105b?version=3.29.4